Styles/Components/toast.css

```diff
@@ -0,0 +1,27 @@
+.toast-container {
+  width: 50%;
+}
+.toast {
+  border: transparent 1px solid;
+  border-radius: 8px;
```

Comment:
It is good that you didnot convert 'px' to 'rem' for small value as it doesn't make much difference.